.github/workflows/release-build-rc.yml

```diff
@@ -0,0 +1,74 @@
+name: Release - Build node release candidate
+
+on:
+  workflow_dispatch:
```

Comment:
is it safe/ok that anyone in org (even accidentally) can run this?

Reply:
This won't be the case, cause the setup is going so that it will be possible to trigger it only from the new paritytech-release org